Dockerfile-lint

```diff
@@ -1,4 +1,4 @@
 FROM golangci/golangci-lint:latest
```

Comment:
we're gonna have limitation on this
we I suggest to copy this in project

Reply:
No, we are using GitHub runners, no restrictions here.